plugin-server/src/main/ingestion-queues/queue.ts

```diff
@@ -1,12 +1,12 @@
 import Piscina from '@posthog/piscina'
 import { PluginEvent, ProcessedPluginEvent } from '@posthog/plugin-scaffold'
 
-import { Hub, PreIngestionEvent, Queue, WorkerMethods } from '../../types'
+import { Hub, PreIngestionEvent, WorkerMethods } from '../../types'
 import { status } from '../../utils/status'
 import { KafkaQueue } from './kafka-queue'
 
 interface Queues {
-    ingestion: Queue | null
+    ingestion: KafkaQueue | null
```

Comment:
Thanks for not removing this. While this is useless, my PR would have some really annoying merge conflicts.